htmd/qm/gaussian.py

```diff
@@ -0,0 +1,171 @@
+# (c) 2015-2017 Acellera Ltd http://www.acellera.com
+# All Rights Reserved
+# Distributed under HTMD Software License Agreement
+# No redistribution in whole or part
+#
+import os
+import numpy as np
+
+from htmd.qm.base import QMBase, QMResult
+
+
+class Gaussian(QMBase):
+    """
+    Class to set up and run QM calculations with Gaussian
+
+   See htmd.qm.Psi4 documentation for more details.
+
+    Examples
+    --------
+
+    Create a Gaussian object
+    >>> from htmd.qm import Gaussian
+    >>> qm = Gaussian()
+    >>> qm # doctest: +ELLIPSIS
+    <htmd.qm.gaussian.Gaussian object at 0x...>
+    """
+
+    @property
+    def _command(self):
+        return 'g09 < g09.in > g09.out 2>&1'
+
+    def _completed(self, directory):
+        return os.path.exists(os.path.join(directory, 'g09.out'))
+
+    def _writeInput(self, directory, iframe):
+
+        with open(os.path.join(directory, 'g09.in'), 'w') as f:
+
+            f.write('%nprocshared = {}\n'.format(self._ncpus))
+            f.write('%mem = {} GB\n\n'.format(self._mem))
+
+            if self.theory == 'HF':
+                dispersion = ''
+            elif self.theory == 'B3LYP':
+                dispersion = ' EmpiricalDispersion=GD3'
+            else:
+                raise NotImplementedError
+
+            line = '# %s/%s NoSymm SCF=tight' % (self.theory, self.basis)
+            line += dispersion
+
+            if self.solvent == 'vacuum':
+                pass
+            elif self.solvent == 'PCM':
+                line += ' SCRF=PCM'
+            else:
+                raise NotImplementedError
+
+            if self.optimize:
+                line += ' Opt=ModRedundant'
+
+            if self.esp_points is not None:
+                line +=  ' Prop=(read,field)'
+
+            line += '\n\n'
+            f.write(line)
+
+            f.write('Mol\n%d %d\n' % (self._charge, self.multiplicity))
+            elements = self._molecule.element
+            coords = self._molecule.coords[:, :, iframe]
+            for element, coord in zip(elements, coords):
+                f.write('    %-2s %10f %10f %10f\n' % (element, coord[0], coord[1], coord[2]))
+            f.write('\n')
+
+            if self._restrained_dihedrals is not None:
+                for dihedral in self._restrained_dihedrals:
+                    f.write('%d %d %d %d F\n' % tuple(dihedral))
+                f.write('\n')
+
+            if self.esp_points is not None:
+                f.write('@grid.dat /N\n')
+
+    def _readOutput(self, directory):
+
+        result = QMResult()
+        result.completed = True
```

Comment:
This parameter does not exist in the `QMResult` class.

Reply:
Fixed 9f231ac